frontend/src/components/UserUpdateRoles.vue

```diff
@@ -218,33 +392,61 @@ export default {
           this.userId,
           this.selectedClient.id,
           rolesToAdd
-        )
+        ),
       ])
         .then(() => {
           this.getUserClientRoles();
           this.$store.commit("alert/setAlert", {
             message: "Roles updated successfully",
-            type: "success"
+            type: "success",
           });
-          window.scrollTo(0, 0);
           //Update list of roles from UserDetails module
-          this.$root.$refs.UserDetails.loadUserRoles();
+          this.close();
+          this.loadUserRoles();
+          this.$root.$refs.UserMailboxAuthorizations.getMailboxClients();
         })
         .catch((error) => {
           this.$store.commit("alert/setAlert", {
             message: "Error updating roles: " + error,
-            type: "error"
+            type: "error",
           });
-          window.scrollTo(0, 0);
-        });
-    }
-  }
+        })
+        .finally(() => {
+          window.scrollTo(0, 0);})    
+    },
+  },
 };
 </script>
 
 <style>
-.right-gutters .col {
-  padding: 10px 12px 10px 0;
+.classForDialog {
```

Comment:
something like updateRolesDialog might be a better name for this
I think it can go in the <style scoped> block with the rest of the css for this component

Reply:
agree with the naming, but placing it on <style scoped> will make them ignore this style